src/modules/flow/iio/iio.json

```diff
@@ -53,7 +53,7 @@
          },
          {
            "data_type": "string",
-           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
+           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. Set to hrtimer:<trigger name> if want to use hrtimer trigger. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
```

Comment:
I'm curious: what is this name of hrtimer? A name of a device that is related to hrtimer? Or an arbitrary name?
If the later, couldn't be this hrtimer trigger be shared by nodes? AFAIU, it's a really fast ticking trigger, with nothing in particular - you can't set its ticking time, for instance. So, it could have no name, right? Just set to `hrtimer` or `:hrtimer` and it's enough: sol-iio would use a shared one.
Of course, I may have misunderstood concept, so please, enlighten me on this.